CONTRIBUTING.md

```diff
@@ -67,6 +67,29 @@ tested by circle using `go test -v -race ./...`. If not, they will need a
 `circle.yml`. Ideally, every repo has a `Makefile` that defines `make test` and
 includes its continuous integration status using a badge in the `README.md`.
 
+We expect tests to use `require` or `assert` rather than `t.Skip` or `t.Fail`,
+unless there is a reason to do otherwise.
+We prefer to use [table driven tests](https://github.com/golang/go/wiki/TableDrivenTests)
+where applicable.
```

Comment:
I'd specify where this is applicable.. I'd hate for somebody to take some nice simple non-table driven tests and attempt to fit them into a table because of this comment. Maybe something along the lines of:
`If a test describes a reasonably complex situation it should be isolated in it's own test function (aka. `TestXxx`. Where testing is fairly simple we prefer the tests to grouped through the use of [table driven tests](https://github.com/golang/go/wiki/TableDrivenTests)`
+where applicable

Reply:
makes sense